sailor/utils/config.py

```diff
@@ -29,7 +29,7 @@ def try_log(exception, msg):
         raise
 
 
-class SailorConfig(namedtuple('SailorConfig', CONFIG_PROPERTIES)):
+class SailorConfig(namedtuple('SailorConfig', CONFIG_PROPERTIES, defaults=(None,) * len(CONFIG_PROPERTIES))):
```

Comment:
so I've checked. you get a cryptic error now when you don't have the corresponding block (predictive_asset_insights) in the config.yml.
`_pai_application_url()`
```
TypeError: 'NoneType' object is not subscriptable
```

So we need to change something there now. I will have a look, and also add tests.

Reply:
Hi. maybe a second related error message, If the names in CONFIG_PROPERTIES and yml file does not match system gives funny error in all classes. Would be nice to catch it earlier